src/main/java/gui/DashboardWindow.java

```diff
@@ -181,13 +182,18 @@ private void jButtonLoadScenarioAction() {
             } else {
 
                 try {
-                    scenarioModel = OntologyLogic.getModelFromOntology(pathToOwlFile, jTextFieldScenarioName.getText());
+//                    scenarioModel = OntologyLogic.getModelFromOntology(pathToOwlFile, jTextFieldScenarioName.getText());
```

Comment:
Ogólnie to mam problem z tym, że pomijamy pathToOwlFile, wiadomo, że na razie ta ścieżka jest taka sama, ale użytkownik może gdzieś mieć inny plik.
Mogę tak zrobić, że jak factory jest do tego pliku co trzeba to przekazuje wywołuje to z factory. A jak zmienił się plik to z pathToOwlFile.

Trochę takiej manipulacji ifami, ale ogólnie chcemy uniknąć tych  kosztownych czasowo operacji w ScenerioReaderze? 
```
        this.ontology = this.ontologyManager.loadOntologyFromOntologyDocument(new File(ontologyPath));
        this.factory = new MyFactory(ontology);
```

Reply:
Tak, można tak spróbować, ogólnie miałem na detektorze błąd z czytaniem scenariusza (na moim wcześniejszym branchu tym v3 działa), tutaj po wybtraniu pliku i wpisaniu numeru miałem informację że nie ma takiego scenariusza w pliku owl. Także możesz jeszcze to sprawdzić u siebie na głównym branchu detector

Reply:
Teraz w commicie wróciłem do używanie tego w taki sposób jak w branchu mateusz v3 i działa, czyta nawet scenariusze tworzone przed chwilą tylko wylicza koszt decyzji na 0

Reply:
Odnośnie tej informacji, że nie ma takiego scenariusza w pliku owl to zakładam, że takie coś dostałeś
![image](https://user-images.githubusercontent.com/43828836/100736073-86a45f00-33d2-11eb-8711-3e8cfcc4baa7.png)

Takie coś dostaje się jak w scenario readerze poleci IllegalArgumentException. Na detectorze teraz testowałem i takie coś nie leci jeśli faktycznie jest już taki scenariusz. Ręcznie przetestowałem dla 4 przypadków "217", "217_scenario", "221", "221_scenario", gdzie 217 było utworzone w poprzedniej sesji 221 w aktualnej. Także to wydaje mi się jest ok

Reply:
Bez sensu napisałem ten pierwszy komentarz, jak w drugim potwierdziłeś, że działa. Ale to teraz mamy pewność, że jest OK :)

Reply:
Odnośnie " tworzone przed chwilą tylko wylicza koszt decyzji na 0" wczoraj byłem przekonany, że na detectorze było to ok, teraz sprawdziłem i faktycznie nie działa tak jak powinno. Nie wszystkich ma zer, ale liczy tylko dla "taking action"
![image](https://user-images.githubusercontent.com/43828836/100736905-b4d66e80-33d3-11eb-9dd6-8f8f92be2996.png)
Jeszcze sprawdzę poprzednie commity,  bo gdzieś wydawało mi się, że to funkcjonowało już